pkg/local_object_storage/sawmill/boltdb.go

```diff
@@ -0,0 +1,521 @@
+package sawmill
+
+import (
+	"bytes"
+	"encoding/binary"
+	"math/rand"
```

Comment:
I think it is a wrong `rand` to use.

Reply:
Actually, no. We don't really care about the predictability and just want to find a random spare ID.

Reply:
Oh, I see. For me the rule of thumb is to avoid `math/rand` unless I do tests, models or local private computations. But you are right, technically it should be okay there.